### PR TITLE
Dev-6210 Node 14 & Dead Packages & CSS Updates

### PR DESCRIPTION
**Pull request by jonhill13**
**High level description:**

N/A

**Technical details:**

> • Node 14

> • Remove dead packages

> • update dead css

**JIRA Ticket:**
[DEV-6210](https://federal-spending-transparency.atlassian.net/browse/DEV-6210)

**Mockup:**
N/A

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [N/A] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [N/A] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [N/A] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [N/A] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [N/A] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [N/A] Design review complete `if applicable`
- [N/A] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: webpack/webpack.common.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ module.exports = {
             'process.env': {
                 ENV: process.env.ENV
                     ? JSON.stringify(process.env.ENV)
-                    : JSON.stringify('dev')
+                    : JSON.stringify('qat')
```
Comment (ebdabbs) [inline webpack/webpack.common.js:132]: Thank you!

**Comment by ebdabbs** [comment]
Also - can #2579 be closed?

File: .travis.yml [hunk 2]
```diff
@@ -15,12 +14,9 @@ env:
 cache:
   directories:
     - node_modules
-before_install:
-  - npm install -g @lhci/cli@0.4.x
 script:
-  - npm run dev
```
[the remark this answers is not in the document]
Comment (jonhill13) [reply .travis.yml:21]: we can keep it. the `lhci:audit` script builds our files against the prod config and runs lighthouse.

So we do not need to build our files again against the dev config. But we could keep it like before where we run dev then travis then lighthouse.

This just skips a build process.

My only concern would be that the tests that we run would hit the prod api I guess.

[the remark this answers is not in the document]
Comment (jonhill13) [reply .travis.yml:21]: As far as the readme goes, you want to add the lhci scripts?

[the remark this answers is not in the document]
Comment (jonhill13) [reply .travis.yml:21]: I updated the readme with the new node version. Did you want to add the lhci scripts too?

**Comment by jonhill13** [comment]
Yeah, we can close #2579. I did not want to to see if anyone else wanted to see if they could get it working.

File: package.json [hunk 3]
```diff
@@ -106,31 +107,23 @@
         "moment": "2.19.3",
         "moment-timezone": "^0.5.4",
         "mousetrap": "^1.6.1",
-        "node-uuid": "^1.4.7",
-        "path-to-regexp": "^1.7.0",
         "prop-types": "^15.7.2",
-        "r-dom": "^2.3.2",
-        "react": "^16.8.6",
-        "react-addons-perf": "15.4.2",
+        "react": "^16.13.1",
```
Comment (ebdabbs) [inline package.json:111]: I think we'll need a corresponding component library PR to update to the same React version. IIRC, we ran into issues when running two different versions?

Comment (jonhill13) [reply package.json:111]: I think and hopefully that is only the latest major version change, this is what our app is running on currently at the time of this PR. The `^` allows for increases in minor versions, so we do not even really need this. I just looked our package lock with the new `qat -> staging` PR and it is at `6.14`
Screen Shot 2021-06-16 at 5 21 18 PM

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonhill13) [reply .travis.yml:21]: removed lighthouse from travis after our discussion in the front end dev meeting

**Review by ebdabbs — APPROVED** [review]

**Pull request merged**